e2e/tests/datasets/export-to-notebook.spec.ts

```diff
@@ -77,8 +79,96 @@ describe('Create Dataset', () => {
 
     // Delete Dataset.
     await dataPage.openTab(TabLabelAlias.Data);
-    await dataPage.openTab(TabLabelAlias.Datasets);
+    await waitWhileLoading(page);
+
+    await dataPage.openTab(TabLabelAlias.Datasets, {waitPageChange: false});
+    await waitWhileLoading(page);
+
     await dataPage.deleteDataset(newDatasetName);
+  });
+
+  /**
+   * Create new Cohort thru Dataset Build page. Cohort built from demographics -> Ethnicity.
+   * Create new Dataset with Cohort, then export to notebook in R language.
+   * Delete Cohort, Dataset, and Notebook.
+   */
+  test('Export dataset to notebook in R programming language', async () => {
+    const workspaceCard = await findWorkspace(page);
+    const workspaceName = await workspaceCard.clickWorkspaceName();
+
+    // Click Add Datasets button
+    const dataPage = new DataPage(page);
+    const datasetBuildPage = await dataPage.clickAddDatasetButton();
+    const cohortBuildPage = await datasetBuildPage.clickAddCohortsButton();
+
+    // Include Participants Group 1: Add Criteria: Ethnicity
+    const group1 = cohortBuildPage.findIncludeParticipantsGroup('Group 1');
+    const modal = await group1.includeEthnicity();
+    await modal.addEthnicity([Ethnicity.HispanicOrLatino, Ethnicity.NotHispanicOrLatino]);
+    await modal.clickFinishButton();
+
+    // Check Group 1 Count.
+    const group1Count = await group1.getGroupCount();
+    const group1CountInt = Number(group1Count.replace(/,/g, ''));
+    expect(group1CountInt).toBeGreaterThan(1);
+    console.log('Include Participants Group 1: ' + group1CountInt);
+
+    // Save new Cohort.
+    const newCohortName = await cohortBuildPage.saveCohortAs();
+    await waitForText(page, 'Cohort Saved Successfully');
+    console.log(`Created Cohort "${newCohortName}"`);
+
+    const cohortActionsPage = new CohortActionsPage(page);
+    await cohortActionsPage.clickCreateDatasetButton();
+
+    await datasetBuildPage.selectCohorts([newCohortName]);
+    await datasetBuildPage.selectConceptSets(['Demographics']);
+    await datasetBuildPage.clickSaveAndAnalyzeButton();
+
+    const newNotebookName = makeRandomName();
+    const saveModal = new DatasetSaveModal(page);
+    const newDatasetName = await saveModal.saveDataset({isExportToNotebook: true, notebookName: newNotebookName, language: Language.R});
```

Comment:
136 characters is really excessive... Is the linter not active here?

In this case I would wrap the object since it's a pretty important part of the test:

```
const newDatasetName = await saveModal.saveDataset({
  isExportToNotebook: true,
  notebookName: newNotebookName,
  language: Language.R
});
```

Reply:
140 char is the max limit.
wrapped and renamed parameter names to be shorter.
```
const newDatasetName = await saveModal.saveDataset({
      exportToNotebook: true,
      notebookName: newNotebookName,
      lang: Language.R
 });
    await waitWhileLoading(page);
```